### DISCUSSION TITLE
remove https://filamentphp.com/ from footer

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#1882] [Help] [answered]
Hi, how are you? how can i remove the https://filamentphp.com/ link from the footer? Thanks in advanced :)

**Comment by danharrin** [comment, marked as answer]
Set `layout.footer.should_show_logo` to `false` in the config file.

**Reply by jcgmalave** [reply]
Thanks!!!! You are a GENIUS!!!
